tinyfl/scorer.py

```diff
@@ -1,18 +1,54 @@
 from tinyfl.model import create_model, test_model
 import numpy as np
+import torch
+from torch import nn
+from torch.utils.data import DataLoader
+from typing import List
 
 
-def _compute_accuracy(weight, testloader):
+def _compute_accuracy(weight: dict, testloader: Dataloader)-> float:
```

Comment:
use ```Mapping[str, Any]``` for weights everywhere
you need to import Mapping from typing as well
also DataLoader has a typo
```suggestion
def _compute_accuracy(weight: Mapping[str, Any], testloader: DataLoader)-> float:
```